proposals/0216-widget-support.md

```diff
@@ -52,7 +52,7 @@ Today in the vehicle, SDL works as a system with a single main display and a sin
 
 #### System
 
-The terminology "system" in this proposal means the system's endpoint is running inside the machine where the app connects to. In classical terms this means SDL core is running in IVI.
+The terminology "system" in this proposal means the system's endpoint, that is running inside the machine where the app connects to. In classical terms this means SDL core, that is running in IVI.
```

Comment:
You can remove both commas.  

The terminology "system" in this proposal means the system's **endpoint that** is running inside the machine where the app connects to. In classical terms this means **SDL core is running** in IVI.

Reply:
I think both variants are correct and I proactively added commas to highlight the main part of the sentence so that's more clear that I mean an endpoint/SDL Core.